packages/wallet-connectors/src/WalletConnect.ts

```diff
@@ -426,8 +426,18 @@ export class WalletConnectConnection implements WalletConnection {
                 });
                 return signature as AccountTransactionSignature; // TODO do proper type check
             }
-            case 'BinaryMessage':
-                throw new Error(`signing 'BinaryMessage' is not yet supported by the mobile wallets`);
+            case 'BinaryMessage': {
+                const params = { schema: msg.schema.value.toString('base64'), data: msg.value.toString('hex') };
```

Comment:
Can you please put the schema&data object into `params.message` instead? This is what I originally asked for, because I thought it would be nice if the params structure for message signing matches the one used with the browser wallet.